README.md

```diff
@@ -34,18 +34,14 @@ change.
 
 ### Conventions
 
-Each entry is of the form:
-```
-* [The Literal Title of The Book](https://a-link-to-the/book/) - an optional comment if the title isn't self-explanatory
-```
-
-The `.editorconfig` file captures basic source files formatting conventions.
-Many editors support this file natively; others (such as VS code) require a plugin, see https://editorconfig.org/.
-
-The default branch is `main`. See [Renaming the default branch from master](https://github.com/github/renaming) for the rationale.
+Basic conventions around source files formatting are captured in the
+`.editorconfig` file. Many editors support that file natively. Others (such as
+VS code) require a plugin, see https://editorconfig.org/.
 
-Commit logs should describe the specific changes made by a commit. See [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/).
```

Comment:
@kw-fn I want to keep in the guidance about commit logs.

Reply:
I'm not sure I see the need for enforcing this, or making possible contributors feel they need to log edits to documentation.

Reply:
It's just a general hygiene factor for git repositories, for documentation or code. I'd prefer to keep it.